debian/control

```diff
@@ -1,14 +1,18 @@
 Source: snuffleupagus
+Section: php
 Priority: optional
 Maintainer: NBS System <snuffleupagus@nbs-system.com>
-Build-Depends: debhelper (>= 9), php7.0-dev | php7.1-dev | php7.2-dev
+Build-Depends: debhelper (>= 9),
```

Comment:
`php-dev` should be enough.

Reply:
Using this Builds Depends you can compile against any php7 dev packages you have installed.

The issue is that when php7.3 will be available, you'll need to add it in the Build Depends.

To avoid that, there is the package php-all-dev https://packages.debian.org/stretch/php-all-dev

>package depending on all supported PHP development packages
>The package currently depends on php7.0-dev.

So it's still php7.0 only.